src/main/haskell/kore/src/Kore/ASTUtils/AlphaCompare.hs

```diff
@@ -0,0 +1,101 @@
+{-|
+Module      : Kore.ASTUtils.AlphaCompare
+Description : Compare
+Copyright   : (c) Runtime Verification, 2018
+License     : NCSA
+Maintainer  : phillip.harris@runtimeverification.com
+Stability   : experimental
+Portability : portable
+-}
+
+module Kore.ASTUtils.AlphaCompare
+    ( alphaCompare
+    ) where
+
+import Data.List
+
+import Kore.AST.Common
+import Kore.ASTUtils.SmartPatterns
+
+alphaCompare
```

Comment:
Could we name this `alphaEq` instead? The name is slightly misleading because `compare` exists, and this does not do what that does.